hw_29_drinks/js/letElemli.js

```diff
@@ -0,0 +1,43 @@
+//#create h2 drinks with id drinks-header
+let drinksHeader = document.createElement("h2");
+drinksHeader.innerHTML = "Drinks";
+drinksHeader.id = "drinks-header";
+document.body.appendChild(drinksHeader);
+//#create array drinks
+let drinks = [
+  "Coca-Cola",
+  "Tea",
+  "Soda",
+  "Milk",
+  "Mineral Water",
+  "Tea",
+  "Coffee",
+  "Milk",
+  "Pepsi",
+  "Coca-Cola",
+  "Coffee",
+];
+//#create ul with id drinks-list
+let drinksList = document.getElementById("drinks-list");
+for (let i = 0; i < drinks.length; i++) {
+  let li = document.createElement("li");
+  li.innerHTML = drinks[i];
+  drinksList.appendChild(li);
+}
+console.log(drinks);
+//#create array uniqueDrinks
+let uniqueDrinks = [];
+for (let i = 0; i < drinks.length; i++) {
+  if (uniqueDrinks.indexOf(drinks[i]) === -1) {
+    uniqueDrinks.push(drinks[i]);
+  }
+}
+uniqueDrinks.sort();
+console.log(uniqueDrinks);
+//#create li for each unique drink
+drinksList.innerHTML = "";
+for (let i = 0; i < uniqueDrinks.length; i++) {
+  let li = document.createElement("li");
+  li.innerHTML = uniqueDrinks[i];
+  drinksList.appendChild(li);
+}
```

Comment:
Not bad, but here also more elegant implementation instead of for loop
```javascript
uniqueDrinks.forEach((value) => {
  drinksList.innerHTML += `<li>${value}</li>`;
});
```

Reply:
Your code works, but you need to do as described in the task (take elements out of html). Let's also agree that such code should not go into production. There should be a single function that describes all the logic, there should be no check consoles. Also your comments in this task are not relevant, although I'm not against comments, but they describe what is clear from the code.